Makefile

```diff
@@ -971,15 +971,9 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
-lint-md-clean:
-	$(RM) -r tools/remark-cli/node_modules
-	$(RM) -r tools/remark-preset-lint-node/node_modules
-
 lint-md-build:
-	if [ ! -d tools/remark-cli/node_modules ]; then \
-		cd tools/remark-cli && ../../$(NODE) ../../$(NPM) install; fi
-	if [ ! -d tools/remark-preset-lint-node/node_modules ]; then \
-		cd tools/remark-preset-lint-node && ../../$(NODE) ../../$(NPM) install; fi
+	if [ "$(OSTYPE)" = darwin -a ! -d tools/remark-cli/node_modules/fsevents/lib ]; then \
```

Comment:
Actually can you change `if` to `@if`, so it doesn't print the output (see https://github.com/nodejs/node/pull/16551).

Reply:
I expect it's not actually necessary to build.  It's only used by Chokidar and that has a fallback.

Reply:
I just removed the line and it's still working. As Ben mentioned, chokidar handles fsevents as an optional dependency. Let's go with it :)